Documentation/using-rkt-with-kubernetes.md

```diff
@@ -10,17 +10,15 @@ This allows a Kubernetes cluster to leverage some of rkt's security features and
 The container runtime is configured at the _kubelet_ level.
 The kubelet is the agent that runs on each machine to manage containers.
 The kubelet provides several flags to use rkt as the container runtime:
-- `--container-runtime=rkt` chooses rkt as the runtime.
-- `--rkt-path` sets the rkt binary path.
-- `--rkt-stage1-image` sets the stage1 image path.
 
-The [getting started with rkt guide][] in the upstream Kubernetes documentation provides more detailed information about how to launch a kubernetes cluster with rkt, how to debug it, and more.
+- `--container-runtime=rkt` sets the node's container runtime to rkt.
+- `--rkt-api-endpoint=HOST:PORT` sets the endpoint of the rkt API service, default to `localhost:15441`.
+- `--rkt-path=PATH_TO_RKT_BINARY` sets the path of the rkt binary. If empty, it will look for rkt in `$PATH`.
+- `--rkt-stage1-image=STAGE1_NAME` sets the name of the stage1 image, e.g. `coreos.com/rkt/stage1-coreos`. If not set, the default one (`coreos.com/rkt/stage1-coreos`) will be used.
 
-[getting started with rkt guide]: http://kubernetes.io/docs/getting-started-guides/rkt/
+More information on how to launch and debug the rktnetes cluster in different environments can be found in the [official user guide](http://kubernetes.io/docs/getting-started-guides/rkt/).
 
 ### Current Status
 
-Integration of rkt as a container runtime for Kubernetes is under active development.
-For the latest information on the progress of the integration, check out [this Google doc][rkt-k8s-checklist] which tracks the detailed status of implemented functionality.
-
-[rkt-k8s-checklist]: https://docs.google.com/document/d/1dYxInIUDTm4HEArQ9Hom_1NhYw22WrXWdglnaLjtQsI/edit
+Integration of rkt as a container runtime for Kubernetes is officially supported and [announced in 1.3 release](http://blog.kubernetes.io/2016/07/rktnetes-brings-rkt-container-engine-to-Kubernetes.html).
+The known issues and notes could be found in the [rktnetes notes](http://kubernetes.io/docs/getting-started-guides/rkt/notes/).
```

Comment:
"Known issues and tips for using rkt with Kubernetes can be found in the [rktnetes notes](http://kubernetes.io/docs/getting-started-guides/rkt/notes/)."